rumbo_logic/src/metrics/cpu.rs

```diff
@@ -4,4 +4,27 @@ use super::prelude::*;
 pub struct CpuUsageInfo {
     core: u8,
     load_percents: u8,
+}
+
+fn get_cpu_usage_info() -> Vec<CpuUsageInfo> {
+    let mut system =
+        System::new_with_specifics(RefreshKind::new().with_cpu(CpuRefreshKind::everything()));
+    std::thread::sleep(System::MINIMUM_CPU_UPDATE_INTERVAL);
+    system.refresh_cpu();
+    let mut i: u8 = 0;
```

Comment:
Instead of using a mutable variable, you could try to implement it with _.enumerate()_ right after _into_iter()_

Reply:
I resolve it in acdb296bbdc61ff808f85eab9848c22186fa9b20